tox.ini

```
@@ -1,29 +1,26 @@
 [tox]
-envlist =
-    py27-test
-    py37-test
-    py27-test-clocale
-    py37-test-clocale
-    py27-test-sphinx17
-    py36-test-sphinx17
-    py37-test-sphinx18
-    py37-test-sphinx20
-    py37-test-sphinxdev
+envlist = py{36,37,38}-test-sphinx{17,18,20,21,22,23,24,30,31,32,dev}{-clocale,}
 requires = pip >= 18.0
            setuptools >= 30.3.0
 
 [testenv]
 changedir = .tmp/{envname}
 deps =
-    sphinx17: sphinx==1.7.9
-    sphinx18: sphinx==1.8.5
-    sphinx20: sphinx==2.0.1
-    sphinx21: sphinx==2.1.2
+    sphinx17: sphinx==1.7.*
+    sphinx18: sphinx==1.8.*
+    sphinx20: sphinx==2.0.*
+    sphinx21: sphinx==2.1.*
+    sphinx22: sphinx==2.2.*
+    sphinx23: sphinx==2.3.*
+    sphinx24: sphinx==2.4.*
+    sphinx30: sphinx==3.0.*
+    sphinx31: sphinx==3.1.*
+    sphinx32: sphinx==3.2.*
     sphinxdev: git+https://github.com/sphinx-doc/sphinx.git
 extras =
     test: test
 commands =
-    test: pytest --pyargs sphinx_automodapi --cov-append --cov sphinx_automodapi --cov-report=
+    test: pytest --pyargs sphinx_automodapi --cov sphinx_automodapi
```

Running coverage for everything seems overkill.
```suggestion
    test: pytest --pyargs sphinx_automodapi
```

---

It's pretty cheap for a small package like this though - any reason why not to do it?

---

We don't seem to be reporting coverage. So, what is the point?

---

Ah of course then I agree! But really we should report coverage, will try and fix in a follow up PR